sdk/identity/identity/src/credentials/defaultAzureCredential.ts

```diff
@@ -260,10 +129,33 @@ export class DefaultAzureCredential extends ChainedTokenCredential {
           // If AZURE_TOKEN_CREDENTIALS is set to "prod", use the production credential chain.
           credentialFunctions = prodCredentialFunctions;
           break;
+        case "environmentcredential":
+          credentialFunctions = [createEnvironmentCredential];
+          break;
+        case "workloadidentitycredential":
+          // If AZURE_TOKEN_CREDENTIALS is set to "workloadidentitycredential", use the WorkloadIdentityCredential.
```

Comment:
I'd love to remove all of these comments, these are pretty self-explanatory

Reply:
done